sdk/tests/wallet/mod.rs

```diff
@@ -20,3 +20,5 @@ mod syncing;
 mod transactions;
 #[allow(clippy::module_inception)]
 mod wallet;
+#[cfg(not(target_os = "windows"))]
```

Comment:
Is that because the database itself would be different?

Reply:
Yes for some reason a db created with linux is broken on windows